src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+.page {
+  margin: 0;
+  padding: 0;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../images/star.svg');
+  background-size: cover;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. According to the checklist, add only top or bottom margins but not both to avoid potential margin collapse issues. Here you have a right margin, which is fine as long as you maintain consistency throughout your project.